api/samples/cbrtrace.c

```diff
@@ -44,19 +44,24 @@
 #include "dr_api.h"
 #include "drmgr.h"
 #include "utils.h"
+#include "assert.h"
 
 static client_id_t client_id;
 
 static int tls_idx;
-
+static long cbrn = 0;
```

Comment:
I have never seen "n" as a suffix -- best to use a clearer name like "cbr_count".